index.js

```diff
@@ -1,3 +1,10 @@
 // play this: https://www.youtube.com/watch?v=d-diB65scQU
 
 // code away!
+
+const server = require("./server");
+
+server.listen(4000, () => {
+    console.log('\n* Server Running on http://localhost:4000 *\n');
+  });
```

Comment:
Server listener looks good but it is also recommended that you use environment variables to make ports dynamic.